code/__DEFINES/traits.dm

```diff
@@ -186,7 +186,8 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_NICE_SHOT			"nice_shot" //hnnnnnnnggggg..... you're pretty good....
 
 //non-mob traits
-#define TRAIT_PARALYSIS			"paralysis" //Used for limb-based paralysis, where replacing the limb will fix it
+#define TRAIT_PARALYSIS				"paralysis" //Used for limb-based paralysis, where replacing the limb will fix it
+#define TRAIT_LIMB_DISABLED_WOUND	"disabledlimb_wound" //Used for limb-based paralysis, where replacing the limb will fix it
```

Comment:
This looks like a source of `TRAIT_PARALYSIS` more than a trait of its own, no?

Reply:
duplicate comment